tests/protos/protos/TestRectangleArena.proto

```diff
@@ -0,0 +1,109 @@
+#VRML_SIM R2020b utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
+# tags: static
+# A configurable rectangle arena composed of a floor surrounded by walls.
+
+PROTO TestRectangleArena [
```

Comment:
I am not sure to understand why we need to copy this PROTO here?

Reply:
When we will apply the modification described in #1867, the `floorAppearance` field of the default RectangleArena Proto won't be any longer a template regeneration field and thus this test will be useless.
So in this case it is better to use a custom project PROTO to be sure that also in the future the correct case will be tested. Note that this test would also change if the depracation message will be removed. 

But I will update this PROTO so that even after #1867 it will remain a regeneration field (currently it is not the case).